plugins/InspectionGadgets/src/inspectionDescriptions/AssertWithSideEffects.html

```diff
@@ -6,7 +6,7 @@
 fields.
 <p>Example:</p>
 <pre>
-  assert i++ < 10;
```

Comment:
Could you keep the 2 space indent? It is nicer in the UI when the code examples are indented a litte.